definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-nonprod.json

```diff
@@ -54,5 +54,40 @@
     "CaseFieldID": "DerivedCoRespondentSolicitorAddr",
     "UserRole": "caseworker-divorce-superuser",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "CoRespondentSolicitorCompany",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "CoRespondentSolicitorCompany",
+    "UserRole": "caseworker-divorce-courtadmin-la",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "CoRespondentSolicitorCompany",
+    "UserRole": "caseworker-divorce-courtadmin",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "CoRespondentSolicitorCompany",
+    "UserRole": "citizen",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "CoRespondentSolicitorCompany",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "CRUD"
```

Comment:
Added the new field's authorisations object for each existing user roles. Please advise.

Reply:
To continue with the same way that was used with the other co-respondent fields, would just add 1 authorisation object with rights = "CRU" for the caseworker_beta.
You can remove the others for now.